src/topology/homotopy/pi_n.lean

```diff
@@ -66,220 +69,129 @@ The first projection of a non-zero dimensional cube
 @[simp] def head {n} : I^(n+1) → I := λ c, c 0
 
 @[continuity] lemma head.continuous {n} : continuous (@head n) := proj_continuous 0
+
 /--
-The last `n` projection of an `n+1` dimensional cube
+The last `n` projections of an `n+1` dimensional cube
```

Comment:
This sounds better:

The projection to the last `n` coordinates from an `n+1` dimensional cube